### PR TITLE
Adding excel to date time function to package 

### PR DESCRIPTION
**Pull request by aylapear**

### CODE REVIEW

File: R/excel_to_date_time.R [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#' dtt_excel_to_date_time(45324.1234, tz = "UTC")
+#' dtt_excel_to_date_time(42370.1234, modern = FALSE)
+dtt_excel_to_date_time <- function(x, tz = dtt_default_tz(), 
+                                   modern = TRUE, ...) {
+  chk::chk_unused(...)
+  chk::chk_flag(modern)
+  chk::chk_numeric(x)
+  chk::chk_string(tz)
+
+  if (length(x) == 0) {
```
Comment (joethorley) [inline R/excel_to_date_time.R:28]: `if(!length(x)) {`

is an alternative

**Review by joethorley — CHANGES_REQUESTED** [review]
Looks good - a couple of super minor suggestions for improvement.

File: R/excel_to_date_time.R [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+#' @export
+#'
+#' @examples
+#' dtt_excel_to_date_time(42370.1234)
+#' dtt_excel_to_date_time(c(1000.1145, 43397.84578))
+#' dtt_excel_to_date_time(45324.1234, tz = "UTC")
+#' dtt_excel_to_date_time(42370.1234, modern = FALSE)
+dtt_excel_to_date_time <- function(x, tz = dtt_default_tz(), 
+                                   modern = TRUE, ...) {
+  chk::chk_unused(...)
```
[the remark this answers is not in the document]
Comment (aylapear) [reply R/excel_to_date_time.R]: Great suggestion, I have updated it to follow the order they appear in the function definition

**Review by joethorley — APPROVED** [review]
looks great

**Pull request merged**